# HG changeset patch
# User Rik <rik@octave.org>
# Date 1454218491 28800
#      Sat Jan 30 21:34:51 2016 -0800
# Node ID 5a02590028c6db620608c63a6d42b94f49fef0bd
# Parent  4c54a269ad44c0a0465c55199ec17eb9ce1a8ac1
doc: Update docstring for nargin.

* ov-usr-fcn.cc (Fnargin): Update docstring.

diff --git a/libinterp/octave-value/ov-usr-fcn.cc b/libinterp/octave-value/ov-usr-fcn.cc
--- a/libinterp/octave-value/ov-usr-fcn.cc
+++ b/libinterp/octave-value/ov-usr-fcn.cc
@@ -785,34 +785,36 @@ DEFUN (nargin, args, ,
 @deftypefn  {} {} nargin ()\n\
 @deftypefnx {} {} nargin (@var{fcn})\n\
 Report the number of input arguments to a function.\n\
 \n\
 Called from within a function, return the number of arguments passed to the\n\
 function.  At the top level, return the number of command line arguments\n\
 passed to Octave.\n\
 \n\
-If called with the optional argument @var{fcn}---a function name or handle---\n\
-return the declared number of arguments that the function can accept.\n\
+If called with the optional argument @var{fcn}---a function name or\n\
+handle---return the declared number of arguments that the function can\n\
+accept.\n\
 \n\
 If the last argument to @var{fcn} is @var{varargin} the returned value is\n\
 negative.  For example, the function @code{union} for sets is declared as\n\
 \n\
 @example\n\
 @group\n\
 function [y, ia, ib] = union (a, b, varargin)\n\
 \n\
 and\n\
 \n\
 nargin (\"union\")\n\
 @result{} -3\n\
 @end group\n\
 @end example\n\
 \n\
-Programming Note: @code{nargin} does not work on built-in functions.\n\
+Programming Note: @code{nargin} does not work on compiled functions\n\
+(@file{.oct} files) such as built-in or dynamically loaded functions.\n\
 @seealso{nargout, narginchk, varargin, inputname}\n\
 @end deftypefn")
 {
   int nargin = args.length ();
 
   if (nargin > 1)
     print_usage ();
 
